videos.js

```diff
@@ -0,0 +1,32 @@
+const videos = {
+    0: "https://www.youtube.com/embed/tw7XBKhZJh4",
+    1: "https://www.youtube.com/embed/CN-_zzHpcdM",
+    3: "https://www.youtube.com/embed/pbU_NzZiEAI",
+    4: "https://www.youtube.com/embed/jZXgrhXJw9o",
+    5: "https://www.youtube.com/embed/XoUBqZZgUEI",
+
```

Comment:
I would suggest saving these links in a separate `.json` file. 

That way, when you want to add more videos, we'll just need to modify the json file instead of the code

Reply:
Ahh good idea! Resolved :) fe787d32a319dd0eb330af376ce30ec82ed307d7